docs/en/ufs/CephFS.md

```diff
@@ -0,0 +1,192 @@
+---
+layout: global
+title: CephFS
```

Comment:
can you modify https://github.com/Alluxio/alluxio/blob/master/docs/en/ufs/Ceph.md to be CephObjectStore to differentiate from CephFS?

Reply:
> can you modify https://github.com/Alluxio/alluxio/blob/master/docs/en/ufs/Ceph.md to be CephObjectStore to differentiate from CephFS?

yes, i am glad to do it.